### ISSUE TITLE
Update to codelyzer v6 or have less strict versioning

### ISSUE THREAD

**Issue by wSedlacek** [ng-seed/angular-tslint-rules#70]
**I'm submitting a ...**  (check one with "x")
```
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  <!-- Please check the repository for a similar issue or PR before submitting -->
[x] Support request => <!-- Please check the repository for a similar issue or PR before submitting -->
[ ] Feature request
[ ] Documentation issue or request
```

**Current behavior**
Currently the following warning is shown when installing.
```
warning "tslint-config-suiyobi > angular-tslint-rules@1.20.4" has incorrect peer dependency "codelyzer@^5.0.0".
```

**Expected/desired behavior**
I would expect `codelyzer@^6.0.0` to be supported 

**Minimal reproduction of the problem with instructions**
Install the package with `codelyzer@^6.0.0` installed in the same project.

**What is the motivation / use case for changing the behavior?**
To avoid false positive warnings when installing.

**Environment**
* **Angular version:** 10.1.0

* **For Tooling issues:**
- Node version: 12.18.1  
- Platform: macOS  

* Others:

https://github.com/fulls1z3/angular-tslint-rules/blob/9c30c2f283bb8e2513886ae24f4db5bde6ffd9bf/package.json#L36
